netlify/edge-functions/github-auth.ts

```diff
@@ -70,10 +70,10 @@ async function getUser(token: string) {
   }
 
   const emails = (await emailResponse.json()) as Array<{ email: string; primary: boolean; visibility: string }>
-  const primaryEmail = emails.find((email) => email.primary && email.visibility === "public")
+  const primaryEmail = emails.find((email) => email.visibility === "public")
```

Comment:
Should we first check if the primary email is public and if it's not public we fallback to the first public email?

Reply:
I think it's not necessary. It would make more complex the logic for looking for email.

Reply:
Ok 👍